docs/conf/prague/2020/schedule.rst

```diff
@@ -81,7 +81,7 @@ We'll also help groups organize dinner plans, so you can continue your conversat
 * **Where**: {{about.venue}}, {{about.mainroom}}
 * **When**: **17:00-20:00**
 
-Monday, May 4
+{{date.day_three.dotw}}, {{date.day_three.date}}
 -------------
```

Comment:
These likely need to be adopted to be dynamic, as an example:

```suggestion
--{{ "-" * date.day_three.dotw|length }}{{ "-" * date.day_three.date|length }}
```

Reply:
@ericholscher should I apply this suggestion? you wrote that it's an example, so I'm not sure? what about other places? maybe @mxsasha can test?

Reply:
I am handling this.